### ISSUE TITLE
Doesn't work on Python 3.7

### ISSUE THREAD

**Issue by numbermaniac** [gawel/irc3#164]
```
File "/usr/local/lib/python3.7/site-packages/irc3/__init__.py", line 132
    if self.config.async:
                       ^
SyntaxError: invalid syntax
```
I'm guessing the solution is similar to [this one](https://github.com/Rapptz/discord.py/commit/1863a1c6636f53592519320a173ec9573c090c0b).

**Comment by numbermaniac** [comment]
Additionally:

```python
File "/usr/local/lib/python3.7/site-packages/irc3/__init__.py", line 14, in <module>
    from . import base
  File "/usr/local/lib/python3.7/site-packages/irc3/base.py", line 74
    async=True,
        ^
SyntaxError: invalid syntax
```

**Comment by gawel** [comment]
There's an open PR: #163
I'll look at it but it'll take some times. I'm on a trip... :)

**Comment by gawel** [comment]
Just released a 1.1.0 with py37 compat. Cheers.
